core/build.gradle.kts

```diff
@@ -161,7 +157,26 @@ kotlin {
             }
         }
     }
+
     sourceSets {
+        groupSourceSets("linux", listOf("linuxX64", "linuxArm64", "linuxArm32Hfp"))
```

Comment:
just in case: probably it could be easier to use KGP hierarchy API for this: [example from kotlinx.atomicfu](https://github.com/Kotlin/kotlinx-atomicfu/blob/b5db8fc8c9f6a85d2ccee8cdf58c68d8a368f7d4/atomicfu/build.gradle.kts#L131-L147), [example from ktor](https://github.com/ktorio/ktor/blob/4fd22824f87ccf30d294675e36168c92e2a97675/build-logic/src/main/kotlin/ktorbuild/targets/KtorTargets.kt#L114-L161)

Reply:
I was wrong. After clearing all caches, `wasmJs` and `wasmWasi` are actually present.